int16 pitch_motor
int16 yaw_motor
